lib/rubocop/cop/minitest/refute_equal.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # Check if your test uses `assert` with negation instead of `refute_equal(expected, actual)`.
+      #
+      # @example
+      #   # bad
+      #   assert(!"rubocop-minitest".eql?(actual))
+      #   assert("rubocop-minitest" != actual)
+      #
+      #   # good
+      #   refute_equal("rubocop-minitest", actual)
+      #
+      class RefuteEqualNil < Cop
+        MSG = 'Prefer using `refute_equal(%<preferred>s)` over ' \
+              '`assert(%<over>s)`.'
+
+
+        def_node_matcher :assert_not_equal_operator, <<~PATTERN
```

Comment:
@koic I tried to isolate the both node_matchers into one but I couldn't.. do you have some idea on how can I do it?

Reply:
I think that can be implemented by `{` ... `}` node pattern. How about that?
https://github.com/rubocop-hq/rubocop/blob/v0.75.0/lib/rubocop/node_pattern.rb#L34

> `'{send class}'      # matches (send ...) or (class ...)`